starters/next-drupal-starter/lib/drupalStateContext.js

```diff
@@ -0,0 +1,76 @@
+import { createContext, useContext } from "react";
+import { DRUPAL_URL } from "./constants";
+import { DrupalState } from "@pantheon-systems/drupal-kit";
+
+const DrupalStateContext = createContext();
+
+// For each locale, make an instance of DrupalState (LocaleStore)
+const makeLocaleStores = (locales, auth = false) =>
+  locales.length > 1
+    ? locales.map(
+        (locale) =>
+          new DrupalState({
+            apiBase: DRUPAL_URL,
+            defaultLocale: locale,
+            debug: true,
```

Comment:
@backlineint  something I've seen other projects do is use env vars for things like this. How do you feel about adding a `DEBUG_MODE` env variable?
This approach could also be used for `DEFAULT_LANGUAGE` and maybe a few other things.

Reply:
> it will also need to be able to run on the client side.

Not sure I understand this part.

Should we allow per-request debugging in DrupalState? As it stands, these stores are created wether or not the `useDsContext` hook is used so I'm not sure how to pass in `debug` as a parameter there.

What if it's a param on `makeLocaleStores` like `auth` is now?

Reply:
You don't understand that part because my comment didn't make any sense :) I was thinking about this from the perspective of DrupalState, but we're in the context of the starter kit here.

I don't think we need to allow per-request debugging at the moment. And yes, adding it as a param to makeLocaleStores like auth seems reasonable for now.